acvm_backend_barretenberg/src/composer.rs

```diff
@@ -1,6 +1,6 @@
 use common::barretenberg_structures::*;
-use common::crs::{CRS, G2};
 
+use crate::crs::{CRS, G2};
```

Comment:
G2 is going away with reference string changes

Reply:
I think let's keep it for now. If changes are required for the new interface changes then we can adjust this at the same time.